pages/infrastructure_providers-validators/running_full_node.md

```diff
@@ -1,73 +1,60 @@
+
+```yaml
+# config.toml
+# Example values from Polkachu for dydx-mainnet-1
+persistent_peers=83c299de2052db247f08422b6592e1383dd7a104@136.243.36.60:23856,1c64b35055d34ff3dd199bb4a5a3ae46b9c10c89@3.114.126.71:26656,3651c82a89f8f4d6fc30fb27b91159f0de092031@202.8.9.134:26656,580ec248de1f41d4e50abe132b7838348db55b80@176.9.144.40:23856,febe75fb6e70a60ce6344b82ff14903bcb53a209@38.122.229.90:26656
 ```
 
-### Getting a Snapshot
-
-See [snapshot service](../infrastructure_providers-network/resources.mdx#snapshot-service).
-
-### Starting a Full Node
-
-Find the seed node's ID and the IP address from [Resources](../infrastructure_providers-network/resources.mdx#seed-nodes). Then, run the following command to start a non-validating full node.
-
-For example,
-```bash
-dydxprotocold start --p2p.seeds="..." --bridge-daemon-eth-rpc-endpoint="<eth rpc endpoint>" --non-validating-full-node=true
-```
-
-💡**Note**: if you want to disable gRPC on your full node, it is important to start the node with the
-`--non-validating-full-node=true` flag. Otherwise, the application will require that gRPC be enabled.
-
-
-## Staying up-to-date with the Latest State
-
-### Connecting to Healthy Peers
-
-In order for the full node to have the latest state, it needs to connect to peers which have the latest state Use the following links to get a list of live peers which have the latest state
-
-https://services.lavenderfive.com/mainnet/dydx#live-peers
-
-https://polkachu.com/live_peers/dydx
-
-Update `persistent_peers`  in the config.toml file to include a randomly selected list of 5 peers from the list of live peers obtained using the links above
-
-### Snapshots
-
-Snapshots contain a compressed copy of the chain data which allow the full node to bootstrap to a recent state in the blockchain. A list of snapshot services can be found [here](../infrastructure_providers-network/resources.mdx#snapshot-service).
-
-### State sync (Alternative to snapshots)
-
-State Sync enables a new node to join the network by obtaining a snapshot of the application state from a state sync node at a recent height. This eliminates the need to fetch and replay all historical blocks. A list of state sync services with instructions are listed below
-
-https://polkachu.com/state_sync/dydx
-
-https://services.lavenderfive.com/mainnet/dydx/statesync
-
-https://autostake.com/networks/dydx/#state-sync
-
-### Address Book
+## Save an Address Book
+Download the latest `addrbook.json` file, which stores configuration details that help your node efficiently connect to peers in its network. Download an up-to-date address book file for your deployment from an [Address Book](../infrastructure_providers-network/resources#address-book-providers) provider.
```

Comment:
1. should we highlight why saving an address book is useful? what can you do with a saved address book?
2. i couldn't find #address-book-providers section. is this in flight?

Reply:
1. I'll look into/ expand on this -- my current understanding is that it is essentially a backup list of healthy peers
2. It should be there, I added it in the same PR. It's a section on the resources page. Double checking the link